lib/interpol/configuration.rb

```diff
@@ -36,12 +36,13 @@ def with_endpoint_matching(method, path)
   # Public: Defines interpol configuration.
   class Configuration
     attr_reader :endpoint_definition_files, :endpoints, :filter_example_data_blocks,
-                :endpoint_definition_merge_key_files
+                :endpoint_definition_merge_key_files, :base_path
     attr_accessor :validation_mode, :documentation_title
```

Comment:
Why not just make `base_path` an `attr_accessor` instead of defining that method below?